frontend/src/hooks/reducers/appStateReducer.ts

```diff
@@ -71,6 +78,7 @@ export type AppStateAction =
   | SaveFieldsAction
   | SaveFarmManureAction
   | SaveImportedManureAction
+  | SaveManureStorageSystemsAction
```

Comment:
I'm not going to ask you to do this because it's a bit complex, but can you make a ticket for me to update the reducer so `saveAnimals` affects `ManureStorageSystems`? Basically, if the generated manure in a storage system gets removed it should also get removed from the storage system, and if a storage system is empty after this it should get fully removed from the array. I can pair-program this and show you what I mean.